### DISCUSSION TITLE
Question about sync wallet with blockchain

### DISCUSSION THREAD

**Discussion by cymqqqq** [RGB-WG/rgb#73] [Q&A]
Hi there, I found the sync wallet with blockchain code in the rgb-lib/wallet/mod.rs, line 1494-1508:
![image](https://github.com/RGB-WG/rgb/assets/8555534/e836bbbf-9764-4b1d-a646-6b262b65e723)
I can't understand the logic, can anyone know about the details write the principal here.

**Comment by dr-orlovsky** [comment]
Wrong place to ask the question: this is not from one of RGB-WG libraries, but from @RGB-Tools by Bitfinex team. It uses BDK which we do not use here.
